prisma/migrations/20221110224543_msats_funcs/migration.sql

```diff
@@ -300,6 +300,7 @@ BEGIN
 
     UPDATE users SET msats = msats - msats_amount - msats_max_fee WHERE id = user_id;
 
+    -- TODO: remove this in a new database migration when we fully migrated from polling to LND subscriptions
```

Comment:
you can't mutate old migrations like this. it changes the hash

Reply:
Ah, I see. `npx prisma migrate dev`:

```
- The migration `20221110224543_msats_funcs` was modified after it was applied.
- The migration `20231201014953_territory` was modified after it was applied.
```